transbank/webpay/webpayplus/webpayplus.py

```diff
@@ -0,0 +1,78 @@
+import requests
+from transbank.webpay.options import *
+
+
+class WPPMeta(type):
+    INTEGRATION_TYPES = dict({
+        "LIVE": "https://webpay3g.transbank.cl/",
+        "TEST": "https://webpay3gint.transbank.cl/",
+        "MOCK": ""
+    })
+
+    _http_client = None
+    _api_key = Options.DEFAULT_API_KEY
+    _commerce_code = Options.DEFAULT_COMMERCE_CODE
+    _integration_type = Options.DEFAULT_INTEGRATION_TYPE
+    _integration_type_url = INTEGRATION_TYPES["TEST"]
+
+    def __init__(self, *args, **kwargs):
+        pass
+
+    # Getters and setters based on the @property decorator
```

Comment:
Is this comment helpful? I guess people familiar with python will know this

Reply:
Sure!